### PR TITLE
UI Improvements & Formatting

### PR DESCRIPTION
**Pull request by RiyaMRoy04**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -14,56 +14,50 @@
...
-    
-        visual_recognition.classify(params, function(err, res) {
-            
-            if (err) {
+    form.parse(req, function (err, fields, files) {
+        if (err) {
+            console.log(err);
+        } else {
+            console.log(fields);
+            const abc = JSON.parse(JSON.stringify(files));
```
Comment (stevemart) [inline server/app.js]: i'd prefer to call this variable something like "file_path", not "abc"

File: server/app.js [hunk 2]
```diff
@@ -14,56 +14,50 @@
...
+            var params = {
+                image_file: fs.createReadStream(abc.myPhoto.path),
+                classifier_ids: ["food"]
```
Comment (stevemart) [inline server/app.js]: "food" should be a const defined earlier, it's a "magic" string right now

File: mobile/www/index.html [hunk 3]
```diff
@@ -33,17 +38,17 @@
             <input type="image" id="captureImage" src="img/camera.png" />
         </header>
         <article>
+            <br/>
+            <div id="cameraInst"><b>Click on the CAMERA ICON to capture image!</b></div>
```
Comment (dolph) [inline mobile/www/index.html]: If that's not obvious, then perhaps the icon needs some tweaking, somehow?

File: mobile/www/index.html [hunk 4]
```diff
@@ -25,6 +25,11 @@
         <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width">
         <link rel="stylesheet" type="text/css" href="css/index.css">
         <title>Calorie Counter</title>
+        <script>
+        setTimeout(function() {
+            $("#cameraInst").remove();
+            }, 2000);
```
Comment (dolph) [inline mobile/www/index.html]: I like this for sure, but would it somehow be possible to take a picture before this disappears? You could add a similar event on-click of the camera icon if so - and possibly drop the delayed remove() until the user takes action first.

File: server/app.js (hunk 1 above)
Comment (dolph) [reply server/app.js]: +1

File: mobile/www/index.html (hunk 4 above)
Comment (RiyaMRoy04) [reply mobile/www/index.html]: @dolph that was just for 2sec but fixed it with onclick event in the dev5 branch commit

**Comment by dolph** [comment]
+1

**Pull request merged**